docs/auth/multi-factor-auth.md

```diff
@@ -21,8 +21,10 @@ instance for the current user. This is the entry point for most multi-factor
 operations:
 
 ```js
-import auth from '@react-native-firebase/auth';
-const multiFactorUser = await auth().multiFactor(auth().currentUser);
+import authModule, { PhoneAuthProvider, getAuth, multiFactor } from '@react-native-firebase/auth';
+
+const auth = getAuth();
+const multiFactorUser = await multiFactor(auth.currentUser);
```

Comment:
this one is going to need some thought as well - apparently `auth.currentUser` just does not exist in modular

It appears that you are supposed to get the user from the returned Credential after sign-in:

https://firebase.google.com/docs/auth/web/password-auth#sign_in_a_user_with_an_email_address_and_password

Or if it was an email link or similar, then `getAdditionalUserInfo` called on the sign-in result will return the user

All of these are likely things we are not doing correctly at the moment

Reply:
auth.currentUser does exist, ignore this, sorry